intelmq/bots/outputs/amqptopic/output.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import sys
+
+from intelmq.lib.bot import Bot
+
+import pika
+
+
+class AMQPTopicBot(Bot):
+
+    CONTENT_TYPE = 'application/json'
```

Comment:
They are only used once, why making a variable from it? Is it planned to extend the bot?